### DISCUSSION TITLE
<title> Github Pages Content

### DISCUSSION THREAD

**Discussion by Kazuhiko-Gushiken** [community/community#78121] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

On the Github Pages ToS, stating what you can and cannot have on your page, it mentioned "violent content and/or activity."

I took that as real content, so: Does this apply to fiction?

Example: Short stories and fake scenarios that may or may not be violent due to the nature of said stories. **Each page would specifically disclaim that all content is fictional and not real in any way shape or form.**

Thank you for your time and answer if you decide to respond.

**Comment by MuchaSsak** [comment]
Well, I found this point in GitHub Docs under Site Policy:
https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies#2-user-safety

So from my understanding, you **must not** include violent/gory imaginary and claim it as "just fiction/cgi"
However, short fictional stories and/or fake scenarios made for entertainment purposes should be fine, I think.

It also depends. For example, if you would've used a story to specifically/directly offend a group of people, or guide them to such violence or whatever. That would not be fine

The most logical thing to me is to just stick to your senses. Your use will probably not violate the policy, as long as you keep in mind not to harass anybody. 👍

**Reply by Kazuhiko-Gushiken** [reply]
Damn, I don't want to risk it since I'm not 100% sure of the consequences. I can't have my account deleted because I've got a lot of stuff I need on here. Thanks anyway!
